### ISSUE TITLE
How can you use a Transformer model as a backbone? No documentation available

### ISSUE THREAD

**Issue by deshwalmahesh** [facebookresearch/detectron2#4505]
Hey,
Just wanted to know that how could you use a pre-trained Transformer model for object detection? I want to use [Microsoft's DiT](https://github.com/microsoft/unilm/tree/master/dit/object_detection). Seems like they have used it with `Detectron2` but no clear steps have been described. Can you please let me know about where can I find the blog / tutorial / documentation on how to fine tune that transformer architecture to use it as a detection backbone with `Faster RCNN` / `Cascade RCNN`.

Thanks in advance :)

**Comment by ZawarK7** [comment]
Hey, 
it has been a long time. Have you implemented the transformer backbone? I am currently using ViT with Faster R-CNN. Let me know if you have implemented it or if you need any help.

**Comment by ae1028** [comment]
> Hey, it has been a long time. Have you implemented the transformer backbone? I am currently using ViT with Faster R-CNN. Let me know if you have implemented it or if you need any help.

Hey. Have you completed your code? I couldn't find anything related to using ViT with Fast RCNN in your account.

**Comment by ZawarK7** [comment]
Yeah,  I have completed the code. Drop your email, and I will share it with you.
